include/reaper/WDL/db2val.h

```diff
@@ -0,0 +1,17 @@
+#ifndef _WDL_DB2VAL_H_
```

Comment:
It looks like this file isn't needed any more?

Reply:
correct. can be kicked out.